src/background.js

```diff
@@ -12,11 +12,12 @@ import Metrics from './lib/metrics';
 import { getAlarms, saveAlarms, removeAlarms,
          getMetricsUUID, getDontShow, setDontShow } from './lib/storage';
 
-const DEBUG = (process.env.NODE_ENV === 'development');
 const WAKE_ALARM_NAME = 'snooze-wake-alarm';
 
 function log(...args) {
-  if (DEBUG) { console.log('SnoozeTabs (BE):', ...args); }  // eslint-disable-line no-console
+  if (process.env.NODE_ENV === 'development') {
+    console.log('SnoozeTabs (BE):', ...args);  // eslint-disable-line no-console
+  }
```

Comment:
So, it's a small function, but we seem to have it in three different places.  What do you think about throwing it in a util file?